kmodes/tests/test_kmodes.py

```diff
@@ -124,6 +124,101 @@
 # Drop target column
 SOYBEAN2 = SOYBEAN2[:, :35]
 
+SOYBEAN4 = np.array([
```

Comment:
@BikashPandey17 , what's the source of this data? This is not the soy bean data (as the name suggests), right?

Reply:
Jaccard dissimilarity was not a good distance metric for the soybean data, as a result, the whole data was considered into a single cluster which looked like this `[0 0 0 0 0 ... 0 0 0]`. I wanted to put an example where distinct clusters were formed so I used this instead. SOYBEAN4 is actually a misleading name since this is just another categorical data with Label encoding(definitely not related to SOYBEAN).

Reply:
Please rename it then. I'll merge after that.